### PR TITLE
Relax ovos-PHAL-plugin-oauth dependency version

### PR DESCRIPTION
**Pull request by Ramblurr**

### CODE REVIEW

**Review by mikejgray — APPROVED** [review]

**Comment by mikejgray** [comment]
@Ramblurr This looks good to me, but I'd like to make sure all the tests pass. Would you please update `.github/workflows/unit_tests.yml` to the following on your branch?

```yaml
name: Unit Tests
on:
  push:
  workflow_dispatch:
  pull_request:
    branches:
      - dev

jobs:
  py_build_tests:
    uses: neongeckocom/.github/.github/workflows/python_build_tests.yml@master
    with:
      python_version: "3.8"
```

**Comment by Ramblurr** [comment]
Done! It needs approval to run: https://github.com/OpenVoiceOS/ovos-PHAL-plugin-homeassistant/actions

**Comment by NeonDaniel** [comment]
Closes #42

File: requirements.txt [hunk 1]
```diff
@@ -4,7 +4,7 @@ ovos-config~=0.0.5
 youtube-search~=2.1
 pytube~=12.1
 websockets~=10.4
-ovos-PHAL-plugin-oauth~=0.0.1
+ovos-PHAL-plugin-oauth>=0.0.1
```
[the remark this answers is not in the document]
Comment (mikejgray) [reply requirements.txt:7]: @Ramblurr This seems like a good alternative, will that work for your use case?

**Review by JarbasAl — APPROVED** [review]

File: requirements.txt (hunk 1 above)
Comment (mikejgray) [inline requirements.txt:7]:
```suggestion
ovos-PHAL-plugin-oauth~=0.0
```

**Review by mikejgray — CHANGES_REQUESTED** [review]

**Comment by mikejgray** [comment]
@Ramblurr This change will be applicable when https://github.com/OpenVoiceOS/ovos-PHAL-plugin-homeassistant/pull/48 is merged, which is slated for this week, so I'm going to close this PR in favor of that one. Thank you for calling this out!

**Pull request closed without merging**